src/structures/GroupChat.js

```diff
@@ -75,6 +75,23 @@ class GroupChat extends Chat {
         }, this.id._serialized, participantIds);
     }
 
+    /**
+     * Set the profile picture of a group chat.
+     * @param {MessageMedia} picture The picture to set. 
+     * @returns {Promise<String | null>} The url of the image / null if you can't set the profile picture
+     */
+    async setProfilePicture(picture) {
+        return await this.client.setProfilePicture(this.id._serialized, picture);
+    }
+
+    /**
+     * Deletes the profile picture of a group chat.
+     * @returns {Promise<Boolean>} true if it succeeded / false if it didn't delete the profile picture.
+     */
+    async deleteProfilePicture() {
+        return await this.client.setProfilePicture(this.id._serialized);
```

Comment:
Should this be calling `deleteProfilePicture` instead of `setProfilePicture`?

Reply:
oops

I usually make PRs locally and test then re-type it out in github's editor.. 

I am going to use git functions from now on, overall less mistakes

Reply:
Thanks for putting an eye on this, sorry for the confusion